DTXManiaプロジェクト/コード/ステージ/04.コンフィグ/CActConfigList.cs

```diff
@@ -215,16 +215,32 @@ public void t項目リストの設定_System()
 				"To save high-scores/skills, turn it ON.\nTurn OFF in case your song data are\n in read-only media (CD-ROM etc).\nNote that the score files also contain\n 'BGM Adjust' parameter. So if you\n want to keep adjusting parameter,\n you need to set SaveScore=ON." );
 			this.list項目リスト.Add( this.iSystemSaveScore );
 
-
-            //this.iSystemChipVolume = new CItemInteger( "ChipVolume", 0, 100, CDTXMania.ConfigIni.n手動再生音量,
-            //    "打音の音量：\n入力に反応して再生されるチップの音\n量を指定します。\n0 ～ 100 % の値が指定可能です。\n",
-            //    "The volumes for chips you hit.\nYou can specify from 0 to 100%." );
-            //this.list項目リスト.Add( this.iSystemChipVolume );
-            //this.iSystemAutoChipVolume = new CItemInteger( "AutoVolume", 0, 100, CDTXMania.ConfigIni.n自動再生音量,
-            //    "自動再生音の音量：\n自動的に再生されるチップの音量を指\n定します。\n0 ～ 100 % の値が指定可能です。\n",
-            //    "The volumes for AUTO chips.\nYou can specify from 0 to 100%." );
-            //this.list項目リスト.Add( this.iSystemAutoChipVolume );
-            //this.iSystemStoicMode = new CItemToggle( "StoicMode", CDTXMania.ConfigIni.bストイックモード,
+		    this.iSystemSoundEffectLevel = new CItemInteger( "Sound Effect Level", CSound.MinimumGroupLevel, CSound.MaximumGroupLevel, CDTXMania.ConfigIni.SoundEffectLevel,
+		        $"[i18n] Sound Effect Level：\n[i18n] The level adjustment for sound effects.\n定します。\n{CSound.MinimumGroupLevel} ～ {CSound.MaximumGroupLevel} % の値が指定可能です。\n", // JDG NEEDS I18N
```

Comment:
At this line, and in the next twenty or so lines that follow, there are some places where I need assistance finalizing text for the config screen and translating it into Japanese.  I would prefer people to reply with translations, so I can update the PR, but repo owners can post new commits if they really want.